src/runner/main.cpp

```diff
@@ -89,6 +91,72 @@ void open_menu_from_another_instance(std::optional<std::string> settings_window)
     PostMessageW(hwnd_main, WM_COMMAND, ID_SETTINGS_MENU_COMMAND, msg);
 }
 
+// since I don't know a better way, we need to watch the keyboard manager config, since it could be changed anywhere
+DWORD WINAPI WatchKeyboardManagerConfigs(LPVOID lpParameter)

```

Comment:
I don't think we should do it here. Please see KeyboardManager.cpp: in `KeyboardManager::KeyboardManager()` we use a dedicated class `EventWaiter` to listen to the main config file changes.